enterprise/cmd/frontend/internal/own/resolvers/resolvers.go

```diff
@@ -181,6 +191,15 @@ func (r *ownResolver) GitBlobOwnership(
 	}
 	total += len(contribResolvers)
 
+	viewerResolvers, err := computeRecentViewSignals(ctx, r.logger, r.db, blob.Path(), repoID)
+	if err != nil {
+		return nil, err
+	}
+	for _, resolver := range viewerResolvers {
+		ownerships = append(ownerships, resolver)
+	}
+	total += len(viewerResolvers)
+
```

Comment:
I think with #51637 this just becomes:

```suggestion
  ownerships = append(ownerships, viewerResolvers...)
```